mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -552,7 +568,19 @@ public void onItemClick(AdapterView<?> adapterView, View view, int i,
 
     @Override
     public void showFetchingError(String s) {
-        Toast.makeText(getActivity(), s, Toast.LENGTH_SHORT).show();
+        if (!Network.isOnline(getContext())) {
```

Comment:
`View` should not contain business logic. This should be controlled with the presenter.